server/service/individualServices/RegardApplication.js

```diff
@@ -75,7 +75,7 @@ exports.regardApplication = function (applicationName, releaseNumber, user, xCor
                         resolve(
                             {
                                 "successfully-connected": false,
-                                "reason-of-failure": "OL_UNKNOWN"
+                                "reason-of-failure": "OL_DID_NOT_REACH_NEW_APPLICATION"
```

Comment:
if (result['status'] != 204) , check whether the status is 408 , then mention that OL did not reach the new application.